engine/src/main/java/org/terasology/rendering/dag/nodes/BlurredAmbientOcclusionNode.java

```diff
@@ -57,31 +56,24 @@
     private static final ResourceUrn SSAO_BLURRED_MATERIAL = new ResourceUrn("engine:prog.ssaoBlur");
     private static final int TEXTURE_SLOT_0 = 0;
 
-    @In
     private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
 
-    @In
-    private WorldRenderer worldRenderer;
-
-    @In
-    private Config config;
-
     private Material ssaoBlurredMaterial;
-    private FBO ssaoBlurredFbo;
     private float outputFboWidth;
     private float outputFboHeight;
 
-    /**
-     * This method must be called once shortly after instantiation to fully initialize the node
-     * and make it ready for rendering.
-     */
-    @Override
-    public void initialise() {
+    @SuppressWarnings("FieldCanBeLocal")
+    private FBO ssaoBlurredFbo;
+
+    public BlurredAmbientOcclusionNode(Context context) {
+        displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
+        Config config = context.get(Config.class);
+
```

Comment:
I used the empty line to separate the context retrieval lines from the rest of the code.